.github/workflows/codespell.yaml

```diff
@@ -0,0 +1,26 @@
+---
+# Referance https://github.com/codespell-project/actions-codespell
+name: Codespell
+# yamllint disable-line rule:truthy
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+    branches:
+      - "master"
```

Comment:
Decide if you need `"master"` with, or without quotes, and use the same everywhere